action_access_control_list/models/ir_protected_action.py

```diff
@@ -1,40 +1,109 @@
 # -*- coding: utf-8 -*-
 # © 2017 Savoir-faire Linux
-# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/gpl).
 
-from odoo import models, fields
+from odoo import models, fields, _
+from odoo.osv.expression import AND, OR, normalize_domain
+from odoo.exceptions import AccessError
+from odoo.tools.safe_eval import safe_eval
 
 
 class IrProtectedAction(models.Model):
-
-    """
-    Secured action
-    """
+    """Secured Action"""
 
     _name = 'ir.protected.action'
```

Comment:
@dufresnedavid IMHO `ir.action.protected` would be a more standard name.

Reply:
@joaoalf I agree